pages/items/index.vue

```diff
@@ -38,31 +39,37 @@
       <div class="text-xs-center pt-2">
         <v-pagination v-model="pagination.page" 
                       :length="pages" 
-                      @next="fetchItems(pagination.page, searchKeyword)" 
-                      @previous="fetchItems(pagination.page, searchKeyword)" 
-                      @input="fetchItems(pagination.page, searchKeyword)" />
+                      @next="fetchItems(pagination.page, searchKeyword, pagination.sortBy || 'code', pagination.descending)" 
+                      @previous="fetchItems(pagination.page, searchKeyword, pagination.sortBy || 'code', pagination.descending)" 
+                      @input="fetchItems(pagination.page, searchKeyword, pagination.sortBy || 'code', pagination.descending)" />
       </div>
     </v-flex>
     <import-modal :show="toggleImportModal" @close="handleCloseModal" />
     <edit-modal v-if="isAnyItemSelected" 
                 :show="isAnyItemSelected" 
                 @close="selectedItem = {}" 
                 :item="selectedItem" />
+    <detail-modal v-model="isShowDetail" :item="detail" />
   </v-layout>
 </template>
 <script>
 import { mapState, mapActions } from 'vuex';
 
 import ImportModal from '@/components/items/ImportModal';
 import EditModal from '@/components/items/EditModal.vue';
+import DetailModal from '@/components/items/DetailModal.vue';
+import _ from 'lodash';
```

Comment:
kalo _import_ lodash nya begini bakal di include semua method nya, padahal yang di butuhin `debounce` doang. lebih efektifnya  bisa jadi `import { debounce } from 'lodash'` . detailnya bisa di lihat di sini : https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash#comment69152391_35251059

Reply:
siap lek

Reply:
tapi ini belum jalan debounce nya. padahal ikut tutorial di vue doc.